testsuite/api/test03.py

```diff
@@ -100,13 +100,14 @@ def printStatus(status, expected):
 ## 		<ssd:Annotations>
 ## 			<ssc:Annotation type="org.openmodelica">
 ## 				<oms:Annotations>
-## 					<oms:SimulationInformation resultFile="test_res.mat" loggingInterval="0.000000" bufferSize="10" signalFilter=".*" />
+## 					<oms:SimulationInformation resultFile="test_res.mat" loggingInterval="0.000000" bufferSize="10" signalFilter="resources/signalFilter.xml" />
 ## 				</oms:Annotations>
 ## 			</ssc:Annotation>
 ## 		</ssd:Annotations>
 ## 	</ssd:DefaultExperiment>
 ## </ssd:SystemStructureDescription>
 ##
+## error:   [getResourceNode] Failed to find node "resources/signalFilter.xml"
```

Comment:
Is this because of the old api `loadSnapshot`?

Reply:
yes this is because of `loadSnapshot()` API, it should be replace with `importSnapshot()`